docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -120,19 +125,54 @@ The Swarm Scheduler is now active. Commands such as ``deis destroy`` or
 To monitor Swarm Scheduler operations, watch the logs of the swarm-manager
 component, or spy on Docker events directly on the swarm-manager machine:
 
+Kubernetes Scheduler
+--------------------
+
+.. important::
+
+    The Kubernetes Scheduler is a technology preview and is not recommended for
+    production use.
+
+`Kubernetes`_ orchestration system for docker containers:
+
+    Kubernetes provides APIs to manage, deploy and scale docker containers. Kubernetes deploys docker containers as `PODS`_,
+    unique entity across a cluster. Containers inside a POD share the same namespaces. More information about `PODS`_.
+
+To run kubernetes on CoreOS requires `flannel`_ which provides subnet to each host for use with containers runtimes.
+Deis repository provides a new user-data file to bootstrap CoreOs machines on the cluster with flannel.
+
+To test the Kubernetes Scheduler, first install and start the kubernetes components:
+
 .. code-block:: console
 
-    $ deisctl journal swarm-manager
-    $ docker -H 172.17.8.102:2395 events
-    2015-04-30T17:31 172.17.8.100:5000/hungry-variable:v5: (from  node:deis-01) pull
-    2015-04-30T17:31 172.17.8.100:5000/hungry-variable:v5: (from  node:deis-02) pull
-    2015-04-30T17:31 02a570: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-01) create
-    2015-04-30T17:31 02a570: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-01) start
-    2015-04-30T17:31 61e59c: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-02) create
-    2015-04-30T17:31 61e59c: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-02) start
+    $deisctl install k8s && deisctl start k8s
+
+Then set the controller's ``schedulerModule`` to "k8s":
+
+.. code-block:: console
+
+    $ deisctl config controller set schedulerModule=k8s
+
+The Kubernetes scheduler is now active. Commands such as ``deis destroy`` or
+``deis scale web=9`` will use `kubernetes ApiServer`_ to manage app PODS.
```

Comment:
The trailing underscore `_` tells ReStructuredText to look for a URL reference with this name, which it didn't find. I'm not sure what was intended here. Please run `make -C docs/ test` to work this out, and please run the test suite in general on this PR before submitting changes.